python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -740,21 +831,32 @@ def __init__(self, schema, peakSchema=None, **kwargs):
     def _addSchemaKeys(self, schema):
         """Add deblender specific keys to the schema
         """
+        # Parent (blend) fields
         self.runtimeKey = schema.addField('deblend_runtime', type=np.float32, doc='runtime in ms')
-
         self.iterKey = schema.addField('deblend_iterations', type=np.int32, doc='iterations to converge')
-
         self.nChildKey = schema.addField('deblend_nChild', type=np.int32,
                                          doc='Number of children this object has (defaults to 0)')
-        self.psfKey = schema.addField('deblend_deblendedAsPsf', type='Flag',
-                                      doc='Deblender thought this source looked like a PSF')
+        self.nPeaksKey = schema.addField("deblend_nPeaks", type=np.int32,
+                                         doc="Number of initial peaks in the blend. "
+                                             "This includes peaks that may have been culled "
+                                             "during deblending or failed to deblend")
+        # Skipped flags
+        self.deblendSkippedKey = schema.addField('deblend_skipped', type='Flag',
+                                                 doc="Deblender skipped this source")
+        self.isolatedParentKey = schema.addField('deblend_isolatedParent', type='Flag',
+                                                 doc='The source has only a single peak '
+                                                     'and was not deblended')
+        self.pseudoKey = schema.addField('deblend_isPseudo', type='Flag',
+                                         doc='The source is identified as a "pseudo" source and '
```

Comment:
Yes, but the `SetPrimaryFlagsTask` technically allows for other types of non-sources, so since it doesn't cost us anything we use that same definition here. This was just a schema key that was reordered, not changed in any way.